src/components/config/robotconfig/PathGradient.tsx

```diff
@@ -87,10 +87,8 @@ class PathGradientFunctions {
     arr: SavedTrajectorySample[],
     documentModel: IStateStore
   ) {
-    const colorValue = Math.abs(
-      Math.sin(point.angularVelocity / 0.00000000001) * 40
-    );
-    return `hsl(${colorValue}, 100%, 50%)`;
+    const normalizedAngularVelocity = (Math.abs(point.angularVelocity) * 100) / 2;
```

Comment:
What are the `* 100` and `/ 2` for?

Reply:
100 is about green ish. 2 is the x max value for angular velocity. higher than that and it shouldn't go more than green.